WalletWasabi/Crypto/ZeroKnowledge/ProofSystem.cs

```diff
@@ -132,7 +132,7 @@ public static Knowledge ZeroProofKnowledge(GroupElement ma, Scalar r)
 			=> new Knowledge(ZeroProofStatement(ma), new ScalarVector(r));
 
 		public static GroupElement PedersenCommitment(Scalar s, Scalar b)
-		 	=> new GroupElement(ECMultContext.Instance.MultBatch(new[] { s, b }, new[] { Generators.Gg.Ge, Generators.Gh.Ge }).ToGroupElement());
+		 	=> new GroupElement(ECMultContext.Instance.MultBatch(null, new[] { s, b }, new[] { Generators.Gg.Ge, Generators.Gh.Ge }, new MultBatchOptions(ECMultiImplementation.Strauss)));
```

Comment:
i don't think this is necessary, because [the default threshold is 88 elements](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Crypto/Groups/GroupElement.cs#L104-L122)

Reply:
After instrumenting the project I saw that it was using peppenger, that's why I put it explicitly.

Reply:
hmm, i saw Strauss used in the flamegraph profile you shared with me and concluded that it's doing the right thing, it's just easy to miss because it's narrow

![image](https://user-images.githubusercontent.com/14242/105496960-1ae5c080-5cb6-11eb-80bb-dd913e4976c8.png)

Reply:
Right. I reverted it.